doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
+    namespace: App\Migrations
+    table_name: migration_versions
+    name: Application Migrations
```

Comment:
What's the purpose of this `name` option? The name is vague, the default value doesn't help me and it's not explained either in the official docs.